apps/sms/style/sms.css

```diff
@@ -933,16 +932,29 @@ body[role="application"] form[role="dialog"][data-type="action"] > header:first-
 }
 
 /* Phone details label styles */
-.phone-type,
+.carrier-information {
+  display: flex;
+  flex-flow: row nowrap; 
+}
+
+.phone-type, .phone-carrier {
+  min-width: 0px;
```

Comment:
ok, actually, I think we need something like a min-width; otherwise if one is very big, both have ellipsis. I think if one is reasonnably small we should try to make it always appear even if one is very big? But min-width doesn't look right, and I don't really the good solution with flex. Maybe with flex we can make it more difficult to shrink than to grow? I'll think about it, I don't have the time to investigate right now, but you can try things on your side as well.